pkg/certificate/manager.go

```diff
@@ -56,6 +56,13 @@ func (m *Manager) Start(checkInterval time.Duration, stop <-chan struct{}) {
 	}()
 }
 
+// GetTrustDomain returns the trust domain from the configured signingkey issuer.
+// Note that the CRD uses a default, so this value will always be set.
+func (m *Manager) GetTrustDomain() string {
+	// TODO(4754): implement
+	return ""
```

Comment:
For my own understanding, will this be something like

```
return m.signingIssuer.TrustDomain
```

Reply:
yup! (with the lock grabbed)

Reply:
Cool thanks!